includes/Core/Modules/Module_With_Screen_Trait.php

```diff
@@ -70,11 +70,17 @@ final public function get_screen() {
 					},
 					'render_callback'     => function ( Context $context ) {
 						$module_info = $this->prepare_info_for_js();
-						?>
-						<script type="text/javascript">var googlesitekitCurrentModule = <?php echo wp_json_encode( $module_info ); ?>;
-						</script>
-						<div id="js-googlesitekit-module" class="googlesitekit-page"></div>
-						<?php
+						$setup_slug = $this->context->input()->filter( INPUT_GET, 'slug', FILTER_SANITIZE_STRING );
+						$reauth = $this->context->input()->filter( INPUT_GET, 'reAuth', FILTER_SANITIZE_STRING );
+						if ( strlen( $setup_slug ) && strlen( $reauth ) ) {
+							?>
+							<div id="js-googlesitekit-module" data-setup-module-slug=<?php echo wp_json_encode( $setup_slug ); ?> data-module-slug="<?php echo wp_json_encode( $module_info['slug'] ); ?> class="googlesitekit-page"></div>
+							<?php
+						} else {
+							?>
+							<div id="js-googlesitekit-module" data-module-slug="<?php echo wp_json_encode( $module_info['slug'] ); ?> class="googlesitekit-page"></div>
+							<?php
+						}
```

Comment:
Same as above here. Also, the closing quote for `data-module-slug=` is missing 🙂

Reply:
https://github.com/google/site-kit-wp/pull/3805/commits/1fef576512f4bcd5bb18504f52d566d13d29b50a